rdkit/Chem/__init__.py

```diff
@@ -43,7 +42,7 @@
   rdCoordGen.SetDefaultTemplateFileDir(templDir)
 
 
-def QuickSmartsMatch(smi, sma, unique=True, display=False):
+def QuickSmartsMatch(smi, sma, unique: bool = True, display: bool = False):
```

Comment:
I leave it to @greglandrum to comment on adding type hints.

Reply:
I am 100% opposed to type hints

Reply:
I'm not going to do that particular rant about why here, but code which uses type hints is inconsistent with the rest of the code base and I won't merge it.

Reply:
OK, I got your idea! I will revert it back to original